hack/test-cmd.sh

```diff
@@ -57,7 +57,7 @@ tests=( $(find_tests ${1:-.*}) )
 
 # test-cmd specific defaults
 TMPDIR="${TMPDIR:-"/tmp"}"
-BASETMPDIR="${BASETMPDIR:-${TMPDIR}/openshift-cmd}"
+export BASETMPDIR="${BASETMPDIR:-${TMPDIR}/openshift-cmd}"
```

Comment:
It requires cleanup, depends on disk IO, might lead to flakiness, and in general test/cmd will give you feedback much later if something got broken.
I normally afford to run unit tests locally before submitting/updating a PR, but not really running test/cmd as part of my workflow (and sure, both groups of tests are run before a PR gets merged).

The other tests we have in test/cmd produce yaml output and grep for expected text in a single line. Specially when we have other ways, I'd strive to keep test/cmd simple.